pkgs/applications/audio/sonobus/default.nix

```diff
@@ -56,6 +56,15 @@ stdenv.mkDerivation rec {
     libXrandr
   ];
 
+  env.NIX_LDFLAGS = lib.optionalString stdenv.hostPlatform.isLinux "-rpath ${lib.makeLibraryPath ([
```

Comment:
I've read discussion in previous MR, and I am still not sure why this is necessary, if it is, then maybe runtimeDependecies no longer necessary ?

Reply:
Good question. I tested, and removing `runtimeDependencies` still breaks the package.

My reason for adding these lines is mostly that I was noticing issues on my laptop (running wayland) that seemed related to these libraries being missing (mostly my cursor randomly not working), and adding these options seemed to fix it. As far as I can tell, there are no negative consequences to including these in the rpath (and these lines are generally included with JUCE apps), so I would like to include them to see if they solve these issues with the package. If nothing changes, I will just remove them in the future. From my testing though, it seems that they do solve the problem.

Reply:
sorry for late reply,  I just tried and without NIX_LDFLAGS -rpath  sonobus doesn't even start for me.

how about we make it use `finalAttrs.runtimeDependencies` ?

Reply:
Thanks for the suggestion. I've gone ahead and made the change. I think the reason why it works on my machine without `NIX_LDFLAGS` is that I have `programs.nix-ld.enable` set to true, so I'll have to remember that in the future.